base/shared/src/main/scala/scalaz/typeclass/SemigroupSyntax.scala

```diff
@@ -6,5 +6,7 @@ import scala.language.experimental.macros
 trait SemigroupSyntax {
   implicit final class ToSemigroupOps[A: Semigroup](a: A) {
     def append(f: => A): A = macro meta.Ops.fa_1
+
+    def <>(f: => A): A = macro meta.Ops.fa_1
```

Comment:
Does this resolve? I don't see how...

After @alexknvl's #1714 it should be easy to get this to work, though; 👍 to the name.

Reply:
It definitely doesn't resolve. I'll delete it.